src/cards/target_similar.ts

```diff
@@ -50,6 +50,11 @@ export function targetSimilarEffect(numberOfTargets: number) {
         // @ts-ignore Find similar units by unitSourceId, find similar pickups by name
         .filter(t => {
           if (isUnit(target) && isUnit(t) && t.unitSourceId == target.unitSourceId) {
+            if (underworld.players.filter(p => !p.isSpawned).map(p => p.unit.id).includes(t.id)) {
+              // Do not allow targeting unspawned players
+              console.log("Filtered out unspawned player from target similar's potential targets");
+              return false;
+            }
```

Comment:
Isn't this redundant, now that addTarget omits unspawned players?

Reply:
Yes, it's redundant. I left it in this spot as well since unspawned players should probably *never* be targeted, and it'll serve as protection against future spells. Not absolutely necessary though, it comes at the cost of passing in underworld.